content/en/core/releases/4.13.0.md

```diff
@@ -0,0 +1,84 @@
+
+---
+title: "4.13.0 release notes"
+linkTitle: "4.13.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.13.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#8119](https://github.com/medic/cht-core/issues/8119): Show confirmation message in training cards when navigating to another page
+
+
+## Highlights
+
+### Targets performance improvements
+
+The calculation and aggregation of targets and recalculation of tasks have been refactored to being calculated automatically when the state changes or documents are updated.
+
+### Training card confirmation modal
+
+A confirmation message is now shown when a user quits the training cards before the training is finished.
+
+{{< figure src="../images/4.13.0-8119.png" >}}
+
+## And more...
+
+### Features
+
+- [#9431](https://github.com/medic/cht-core/issues/9431): Improve accuracy of target document after interval turnover
```

Comment:
This is a big deal - let's add a paragraph explaining why so people actually realise why they should update. In fact this feature is the actual highlight. Instead of talking about performance improvements (which we haven't proven by the way) let's talk about improved accuracy of the target aggregate accuracy!